### ISSUE TITLE
[MHV SM / Accessibility] Move Success alert below H1

### ISSUE THREAD

**Issue by thomsonespy** [department-of-veterans-affairs/va.gov-team#130619]
## Description
After a message is sent, the success alert appears above the page heading (`<h1>`). It should be moved below the page heading. 

## Context

1. The focus management practice on MHV is to [set focus to the main page heading (`<h1>`) when a page is loaded](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/accessibility/documentation/decision-records.md#focus-management-on-page-load). When an alert is positioned before the page heading, users must backtrack to get to the alert and benefit from the alert content and functionality.
2. For wayfinding, the alert relates to the content of the page. Placing the alert before the heading removes the alert from the context provided by the page heading. The level one heading (`<h1>`) should label the page and all content associated with that heading should follow the heading visually and programmatically, including alerts.

## User story

As a Veteran, I need alerts associated with page content to appear after the page heading, so that I know that the content and functionality contained in the alert relates to the page I'm on.

## Resources

- [Link users to sent folder in confirmation alert #124122](https://github.com/department-of-veterans-affairs/va.gov-team/issues/124122) with context in the comments

## Notes

- **Alert placement:** "In most cases, the standard Alert (in all of its variations) should be placed directly below the intro text, near the top of the page." See [VADS guidance on placement of Alerts](https://design.va.gov/components/alert/#web-4)
- **Alert role and screen reader notification:** "Advisory information, not important enough to have an alert role: Use `role="status"` Use this role on alert components that appear after a user interaction. This allows users with assistive tech to be notified of the change, but won’t immediately interrupt them from the current task." See [VADS guidance on assigning an aria-role to alerts](https://design.va.gov/components/alert/#assign-an-appropriate-aria-role)
- **Focus management:** "Set focus to the level 1 heading (`<h1>`) on page load." See [Focus Management on Page Load design decision](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/accessibility/documentation/decision-records.md#focus-management-on-page-load)

## Possible tasks:
- [ ] Update alert placement to after the page heading
- [ ] Use the default focus management (place focus on `<h1>` on page load)
- [ ] Use `role="status"` to announce alert contents 

## Acceptance criteria
- [ ] UX review — alert follows page heading
- [ ] Accessibility review — focus is set to page heading and alert content are announced

**Comment by sarahhortonadhoc** [comment]
@DanielleThierryUSDSVA would you be able to provide a view on positioning feedback alerts like this one? We are proposing to position them consistently, after the page heading. But the [VADS content guidance](https://design.va.gov/content-style-guide/error-messages/feedback#task-completion-successfailure) specifies positioning them before the page heading. 

For context, MHV has a convention of always placing focus on the page heading. We are concerned that positioning an alert before the page heading means people need to backtrack to get to the alert.

**Comment by sarahhortonadhoc** [comment]
Moving from blocked — confirming VADS guidance to position the alert after the page heading based on [updated guidance in the content style guide](https://design.va.gov/content-style-guide/error-messages/) and [Slack thread](https://dsva.slack.com/archives/C01K37HRUAH/p1770155791947329). cc @thomsonespy @Tonksthebear
